extensions/amp-sidebar/1.0/toolbar.js

```diff
@@ -104,7 +101,7 @@ export class Toolbar {
    */
   buildCallback_() {
     this.toolbarClone_ = this.toolbarDOMElement_.cloneNode(true);
-    // Addend "_toolbar" to ids on the toolbar clone
+    // Added "_toolbar" to ids on the toolbar clone
```

Comment:
actually can we just drop the ids and create a bug for it - this is way more complicated than i thought it would be .